types/tests/PermissibleRender.test.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import { render, } from 'react-dom'
+import { PermissibleRender, PermissibleRenderProps, } from '@brainhubeu/react-permissible'
+
+const permissibleRenderTestProps: PermissibleRenderProps = {
+  oneperm: false,
+  children: 'restricted content',
+  userPermissions: ['view'],
```

Comment:
Please change the permission to `CAMEL_CASE` to make the code style similar to the one we're using already.

Reply:
Done.